### ISSUE TITLE
--help is not implemented

### ISSUE THREAD

**Issue by vandenoever** [edolstra/nix-repl#10]
```
$ nix-repl --help
error: unrecognised option ‘--help’
Try ‘nix-repl --help’ for more information.
```
This is true for nix-repl 1.8 and 1.9.

**Comment by benley** [comment]
this can probably be closed since 57aeef0b6a3d3c9506e35f57f5b6db33019967e5 fixes it

**Comment by fuzzy-id** [comment]
I don't think that this should be closed. To implement a help option it is not sufficient to provide something like the following:
```
[root@nixos:/boot]# nix-repl --version
nix-repl (Nix) 1.10

[root@nixos:/boot]# nix-repl --help
Usage: nix-repl [--help|--version]
```
1. There is no help text provided what this program is about.
2. There is no help provided for the options. 
3. Usage description is wrong. You can at least also call `nix-repl '<nixos>'`. So `nix-repl [--help|--version|SOMETHING]` should be more appropriate. I don't know about the type of `SOMETHING` though.
